gatsby_fp/src/components/fieldComponents/profService.js

```diff
@@ -0,0 +1,87 @@
+import React from "react"
+
+export default function ProfService ({profServiceList}){
+  return(
+    
+    <>
+    <div className="sectiontitle">Professional Service</div>
+        <div>
+          <ul>
+          {profServiceList.map((element, index) => (
+            // <li key={index} dangerouslySetInnerHTML ={{__html: 
+            //   `${element.type} - ${element.type_other}<br>${element.elecapp}<br>${element.title} 
+            //   ${element.org} in ${element.city}, ${element.state} (${element.dtm_start} ${element.dty_start} - 
+            //   ${element.dtm_end} ${element.dty_end}) <br> Compensated: 
+            //   ${element.compensated}
+            //   <br>Audience: ${element.audience}`}}>
+            // </li>
+              <li key={index} dangerouslySetInnerHTML ={{__html:` 
+               ${element.type ? `
+                  type: ${element.type}<br> `:
+                  ``
+                }
+                ${element.type_other ? `
+                  type_other: ${element.type_other}<br> `:
+                  ``
+                }
+                ${element.title ? `
+                  title: ${element.title}<br> `:
+                  ``
+                }
+                ${element.org ? `
+                  org: ${element.org}<br> `:
+                  ``
+                } 
+                ${element.dtm_start ? `
+                  dtm_start: ${element.dtm_start}<br> `:
+                  ``
+                } 
+                ${element.dty_start ? `
+                  dty_start: ${element.dty_start}<br> `:
+                  ``
+                }
+                ${element.dtm_end ? `
+                  dtm_end: ${element.dtm_end}<br> `:
+                  ``
+                }  
+                ${element.dty_end ? `
+                  dty_end: ${element.dty_end}<br> `:
+                  ``
+                }
+                ${element.compensated ? `
+                  compensated: ${element.compensated}<br> `:
+                  ``
+                }
+                ${element.desc ? `
+                  desc: ${element.desc}<br> `:
+                  ``
+                }
+                ${element.elecapp ? `
+                  elecapp: ${element.elecapp}<br> `:
+                  ``
+                }
+                ${element.exofficio ? `
+                  exofficio: ${element.exofficio}<br> `:
+                  ``
+                }
+                ${element.city ? `
+                  city: ${element.city}<br> `:
+                  ``
+                }
+                ${element.state ? `
+                  state: ${element.state}<br> `:
+                  ``
+                }
+                ${element.country ? `
```

Comment:
These all hold a similar shape, with similar rendering logic. Perhaps we can loop through the properties of the object, and render those that are set?